recipe/meta.yaml

```diff
@@ -1,34 +1,32 @@
 {% set name = "conda-package-handling" %}
-{% set version = "1.9.0" %}
+{% set version = "2.0.2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://github.com/conda/conda-package-handling/archive/{{ version }}.tar.gz
-  sha256: 81e0771b8b10ab5c20678a5da714e36d97e321acfc3e5b304ec2e650ce84734b
+  sha256: ec0a2c51f442d98b935ee2288beb1ec6c7e25020b5961f8d09f35738b276b532
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
     - cph = conda_package_handling.cli:main
 
 requirements:
   build:
-    - {{ compiler('c') }}
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - python
+    - python >=3.7
     - pip
-    - cython
-    - _libarchive_static_for_cph ==3.6.1
-    - msinttypes  # [win and vc<14]
-    - setuptools
```

Comment:
We still use `setuptools` for building a package, see https://github.com/conda/conda-package-handling/blob/3c90a2e751a2babed46701bb9c972b7b3b474650/setup.py#L4